topology/probes/netlink.go

```diff
@@ -346,6 +346,32 @@ func (u *NetNsNetLinkProbe) addLinkToTopology(link netlink.Link) {
 		metadata["FDB"] = neighbors
 	}
 
+	routeList, err := u.handle.RouteList(link, netlink.FAMILY_ALL)
+	if err == nil && len(routeList) > 0 {
+		routes := make([]map[string]interface{}, len(routeList))
+		for i, route := range routeList {
+			gw := ""
+			if len(route.MultiPath) > 0 {
+				gw = fmt.Sprintf("%s", route.MultiPath)
+			} else {
+				gw = route.Gw.String()
+			}
+			dst := ""
+			if route.MPLSDst != nil {
+				dst = fmt.Sprintf("%d", route.MPLSDst)
+			} else {
+				dst = fmt.Sprintf("%s", route.Dst)
+			}
+			routes[i] = map[string]interface{}{
+				"Src":   route.Src.String(),
+				"Dst":   dst,
+				"GW":    gw,
+				"Table": int64(route.Table),
+			}
+		}
+		metadata["Routes"] = routes
+	}
+
```

Comment:
The code looks fine. 
Just a question, do we have unit tests for the netlink stuff? If we do, I think it will be a good idea to add some Unit test for this feature too.